package.json

```diff
@@ -7,8 +7,8 @@
   "author": "Marlon Keating <marlonakeating@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "webpack --config webpack.config.js && python manage.py collectstatic --noinput",
-    "sass": "node-sass civictechprojects/static/css/styles.scss -o civictechprojects/static/css/compiled --output-style compressed",
+    "build": "webpack --config webpack.config.js && npm run build:sass && python manage.py collectstatic --noinput",
```

Comment:
I read a lot of conflicting positions on whether to use webpack or npm scripts to build the .css from sass partials and then if webpack, _how_ to configure it. While I investigate that situation further I modified the npm run build script to compile the css for now so it's one step instead of two and requires no changes to the developer workflow in terms of commands run.